sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/WebPubSubRestClient.cs

```diff
@@ -355,7 +355,7 @@ public Response CloseClientConnection(string hub, string connectionId, string re
             }
         }
 
-        internal HttpMessage CreateSendToConnectionRequest(string hub, string connectionId, ContentType contentType, RequestContent message)
+        internal HttpMessage CreateSendToConnectionRequest(string hub, string connectionId, Models.ContentType contentType, RequestContent message)
```

Comment:
This is unrelated to this PR, but since now you have a dependency on Azure.Core.Experimental, maybe we should use ContentType from it.

Reply:
Not opposed to it, I suspect we'll be on AZE for a bit.